### PR TITLE
Add function to set URI

### PR DESCRIPTION
**Pull request by hswopeams**
Also updates default uri in deployment script

### CODE REVIEW

**Review by bidyut-arianelabs — DISMISSED** [review]
Looks good to me 👍

File: contracts/ERC1155NonTransferable.sol [hunk 1]
```diff
@@ -122,4 +125,14 @@ contract ERC1155NonTransferable is
...
         _unpause();
     }
+
+    /**
+     * @notice Setting the metadata uri
+     * @param _newUri   New uri to be used
+     */
+    function setUri(string memory _newUri) external onlyOwner {
+        require(bytes(_newUri).length != 0, "INVALID_VALUE");
+        super._setURI(_newUri);
```
Comment (zajck) [inline contracts/ERC1155NonTransferable.sol:135]: `super` is not really needed here.

Comment (hswopeams) [reply contracts/ERC1155NonTransferable.sol]: Removed

**Review by zajck — APPROVED** [review]

**Review by bidyut-arianelabs — APPROVED** [review]
Looks good to me 👍

**Pull request merged**